tracer/src/Datadog.Trace/Vendors/Datadog.Sketches/Stores/DenseStore.cs

```diff
@@ -74,7 +74,9 @@ protected DenseStore(DenseStore store)
 
         if (store.Counts.Length > 0 && !store.IsEmpty())
         {
-            Counts = new double[store.MaxIndex - store.MinIndex + 1];
+            // This change seems correct based on the code in the java repo: https://github.com/DataDog/sketches-java/blob/5031ddff8e6d6de5174d2c06e2ced618f3a48621/src/main/java/com/datadoghq/sketch/ddsketch/store/DenseStore.java#L60-L64
```

Comment:
@kevingosse what do you think? I haven't noticed any particular issues stemming from this code, but I noticed a discrepancy when reviewing this code